src/libbson/src/bson/bson-clock.c

```diff
@@ -98,6 +98,32 @@ bson_gettimeofday (struct timeval *tv) /* OUT */
 }
 
 
+/*
+ *--------------------------------------------------------------------------
+ *
+ * bson_get_real_time_ms --
+ *
+ * Returns:
+ *       The current number of milliseconds that have elapsed since the
+ *       Unix Epoch, or -1 on error
+ *
+ * Side effects:
+ *       None.
+ *
+ *--------------------------------------------------------------------------
+ */
+int64_t
+bson_get_real_time_ms (void)
```

Comment:
I prefer not adding new API unless there is a known use case. We need to maintain the API until a future major release.

The C driver has two consumers: other wrapping drivers, and direct users. If this API helps either, I would be in favor of it. If not, I suggest moving this to internal API within libmongoc. This could be in mongoc-util-private.h or another (even new) private header.

If we do keep it, we should add RST documentation for the new symbol.